.prettierrc.yml

```diff
@@ -0,0 +1,3 @@
+printWidth: 100
+singleQuote: true
+trailingComma: es6
```

Comment:
This could be moved to package.json 🤭

Reply:
I tried to keep as much config out of the package.json :P